FlyWireAnnotationTutorial.ipynb

```diff
@@ -0,0 +1,309 @@
+    "                                                 \"tag\": random_tags,\n",
+    "                                                 \"id\": np.arange(100, 110)})\n",
+    "                                                 \n",
+    "client.annotation.update_annotation_df(table_name=random_table_name,\n",
+    "                                       df=random_annotation_data,\n",
+    "                                       position_columns=[\"pt_position\"])\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reading annotaitons"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Annotations can be read directly from the annotaition service. However, no segment IDs will be assigned to the annotations through this interface. To access annotaions with segment IDs assigned, one must use the [materialization interface](https://github.com/seung-lab/CAVEclient/blob/master/FlyWireSynapseTutorial.ipynb).\n",
+    "\n",
+    "Annotations can be read by ID. Here, we use the nucleus table (`nuclei_v1`, see it online [here](https://prod.flywire-daf.com/annotation/views/aligned_volume/fafb_seung_alignment_v0/table/nuclei_v1)) as example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[{'volume': 6.65092096,\n",
+       "  'bb_start_position': [718496, 261792, 114640],\n",
+       "  'valid': True,\n",
+       "  'bb_end_position': [722016, 265120, 116200],\n",
+       "  'pt_position': [720000, 263360, 115520],\n",
+       "  'superceded_id': None,\n",
+       "  'created': '2021-06-23 19:55:35.166396',\n",
+       "  'deleted': 'None',\n",
+       "  'id': 7415718},\n",
+       " {'volume': 11.52339968,\n",
+       "  'bb_start_position': [708800, 262112, 128200],\n",
+       "  'valid': True,\n",
+       "  'bb_end_position': [712064, 264832, 131080],\n",
+       "  'pt_position': [710592, 263392, 129800],\n",
+       "  'superceded_id': None,\n",
+       "  'created': '2021-06-23 19:55:35.160196',\n",
+       "  'deleted': 'None',\n",
+       "  'id': 7416439}]"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client.annotation.get_annotation(table_name=\"nuclei_v1\",\n",
+    "                                 annotation_ids=[7416439, 7415718])"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
I think your missing an explanation about what the user can expect to happen after they post the table, and how to query it.